### DISCUSSION TITLE
Special Allocations for Exceptional Contributions

### DISCUSSION THREAD

**Discussion by AbdelStark** [keep-starknet-strange/unruggable.meme#99] [Announcements]
# Special Allocations for Exceptional Contributions

In the pursuit of rapid development and maximum security for the MVP of Unruggable Meme, we are transparently disclosing special allocations for exceptional contributions. The urgency for a safer memecoin platform, underscored by recent events, necessitates these expedited efforts.

### Smart Contract Implementation Lead

| **Contributor** | **Allocation ($) USD** | **STRK Tokens** | **Role Description** |
|-----------------|------------------------|-----------------|----------------------|
| @enitrat        | 7,500                  | 5,000           | Leading the smart contract development for the MVP. Responsibilities include coordinating contributions, orchestrating project tasks, and conducting rigorous reviews and project management to meet deadlines. |

### Front-End Implementation Lead

| **Contributor** | **Allocation ($) USD** | **STRK Tokens** | **Role Description** |
|-----------------|------------------------|-----------------|----------------------|
| @0xChqrles      | 5,000                  | 1,500           | Overseeing the front-end development for the MVP. This role entails managing contributions, guiding project execution, reviewing pull requests, and ensuring timely project delivery. |

### Independent Security Audit (1)

| **Contributor** | **Allocation ($) USD** | **STRK Tokens** | **Role Description** |
|-----------------|------------------------|-----------------|----------------------|
| @ermvrs         | 5,000                  | 2,500           | Conducting an independent security audit of the smart contracts. Deliverables include a comprehensive report detailing vulnerabilities, best practices, and actionable recommendations. |

### Independent Security Audit (2)

| **Contributor** | **Allocation ($) USD** | **STRK Tokens** | **Role Description** |
|-----------------|------------------------|-----------------|----------------------|
| @0xEniotna      | 5,000                  | 2,500           | Performing a thorough independent security review of the smart contracts. Expected to provide a detailed analysis report covering identified issues, best practices, and strategic guidance. |

### Independent Security Audit (3)

| **Contributor** | **Allocation ($) USD** | **STRK Tokens** | **Role Description** |
|-----------------|------------------------|-----------------|----------------------|
| @credence0x       | 2,500                  | 1,000           | Performing a thorough independent security review of the smart contracts. Expected to provide a detailed analysis report covering identified issues, best practices, and strategic guidance. |

Each role is critical to the expedited and secure delivery of the Unruggable Meme MVP. These allocations are a testament to our commitment to transparency and excellence in the development process.

**Comment by Cryptobleezy** [comment]
Been trying to get on a project on onlydust for a while now but I've not been able to. Congratulations to the first set of winners.

**Comment by vandehos** [comment]
Great

**Comment by vahidnfc33** [comment]
perfect  :)
